owncloudApp/src/main/res/values-ar/strings.xml

```diff
@@ -476,8 +476,7 @@
   <string name="manage_space_description">سيتم حذف الإعدادات وقواعد البيانات وشهادات الخادم من بيانات %1$s\'s نهائيًا. \n\nسيتم الاحتفاظ بالملفات التي تم تنزيلها دون تغيير.\n\nيمكن أن تستغرق هذه العملية بعض الوقت.</string>
   <string name="manage_space_clear_data">مسح البيانات</string>
   <string name="manage_space_error">تعذّر حذف بعض الملفات.</string>
-  <string name="permission_storage_access">الأذونات الإضافية مطلوبة لتحميل الملفات وتنزيلها.</string>
-  <string name="local_file_not_found_toast">لم يتم العثور على الملف في نظام الملفات المحلية</string>
+    <string name="local_file_not_found_toast">لم يتم العثور على الملف في نظام الملفات المحلية</string>
```

Comment:
It seems that the resource manager in Android Studio adds an indentation when you remove a string resource. This has happened in every file 😅 . Try to reformat the code selecting them all, and if you are not able to, I will approve it 👍